scripts/validate-generated.sh

```diff
@@ -1,35 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Again, this would ideally be removed as part of a separate "remove OpenShift support" PR that I am itching to write.